bundles/core/src/test/resources/teaser/exporter-teaser11.json

```diff
@@ -13,7 +13,7 @@
       "dataLayer": {
         "teaser-84f85e5d32-cta-4a426bc86e": {
           "xdm:linkURL": "/core/content/teasers.html",
-          "@type": "nt:unstructured",
+          "@type": "core/wcm/components/teaser/v1/teaser/item",
```

Comment:
@gabrielwalt Do we want the CTA type to be `<teaser-resource-type>/item` or `<teaser-resource-type>/cta`?

Reply:
`cta` is more self-explanatory, that might be better.

Reply:
changed teaser action item to /cta in https://github.com/adobe/aem-core-wcm-components/pull/1184/commits/140f7f7543877bb2f33614aef6f672d8ab1663c3